docs/architecture_overview/master13-deployment.adoc

```diff
@@ -10,7 +10,7 @@ tiers are as follows.
 . _persistence_: data storage and retrieval.
 . back-end services: including EHR, demographics, terminology, archetypes, security, record location, and so on. In this layer, the separation of the different services is transparent, and each service has a coarse-grained service interface.
 . _virtual EHR_: this tier is the middleware, and consists of a coherent set of APIs to the various back-end services providing access to the relevant services, thereby allowing user access to the EHR; including EHR, demographics, security, terminology, and archetype services. It also contains an archetype- and template-enabled kernel, the component responsible for creating and processing archetype-enabled data. In this tier, the separation of backend services is hidden, only the functionality is exposed. Other virtual clients are possible, consisting of APIs for other combinations of back-end services.
-. _application logic_: this tier consists of whatever logic is specific to an application, which might be a user application, or another service such as a query engine.
+. _application logic_: this tier consists of whatever logic is specific to an application, which might be an user application, or another service such as a query engine.
```

Comment:
Was just about to merge, but this change is not correct because 'user' acts like it was spelled 'yuser', so no liaison needed. If you can reverse this one, we can merge the rest.

Reply:
Sorry. I have reverted this change and updated the commit accordingly. It is not included in the pull request anymore.